controllers/burgers_controller.js

```diff
@@ -27,7 +27,7 @@ router.post("/api/burgers", function(req, res){
 router.put("/api/burgers/:id", function(req, res){
     var iD = "id = " + req.params.id;
 
-    burger.updateOne({devoured: req.body.devoured}, iD, function(result){
+    burger.updateOne({devoured: !!req.body.devoured}, iD, function(result){
```

Comment:
Here the devoured has to be true or false, not 0 or 1.  The double bang insures that it's a boolean.